src/Jaeger.Core/Reporters/RemoteReporter.cs

```diff
@@ -64,7 +67,7 @@ public async Task CloseAsync(CancellationToken cancellationToken)
         {
             // Note: Java creates a CloseCommand but we have CompleteAdding() in C# so we don't need the command.
```

Comment:
Comment refers to old CompleteAdding() instead of the new Complete() method

Reply:
removed